.github/workflows/build.yml

```diff
@@ -45,10 +73,50 @@ jobs:
             ${{ runner.os }}-sbt-${{ hashFiles('project/build.properties') }}-${{ hashFiles('project/plugins.sbt') }}
           restore-keys: ${{ runner.os }}-sbt-
           
-      - name: Run Tests for Java ${{ matrix.java }}, Scala ${{ matrix.scala }} / ${{matrix.platform }}
+      - name: Run Tests for Java ${{ matrix.java }}, Scala ${{ matrix.scala }}, ScalaJS ${{ matrix.scalajs }}
+        run: |
+          ./.github/scripts/build
+        env:
+          SCALA_VERSION: ${{ matrix.scala }}
+          SCALAJS_VERSION: ${{ matrix.scalaJS }}
+          SBT_COMMAND: ci-js
+  
+  mima:
+    name: Mima binary compatibility test
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        java: [ 8 ]
+        scala: [ 2.12.11 ]
+        scalajs: [ 0.6.32 ]
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: olafurpg/setup-scala@v7
+        with:
+          java-version: "adopt@1.${{ matrix.java }}"
+     
+      - name: Run Tests for Java ${{ matrix.java }}, Scala ${{ matrix.scala }}, ScalaJS ${{ matrix.scalajs }}
         run: |
           ./.github/scripts/build
         env:
           SCALA_VERSION: ${{ matrix.scala }}
           SCALAJS_VERSION: ${{ matrix.scalaJS }}
-          PLATFORM: ${{ matrix.platform }}
+          SBT_COMMAND: ci-jvm-mima
+      
+  publish:
```

Comment:
Does it run release after each commit to master?

Reply:
yup - each commit to master for any fork but without the right credentials it should do nothing

Reply:
Cool, automated snapshots could be nice for many users waiting for their fixes